ibis/backends/base/sqlglot/datatypes.py

```diff
@@ -352,6 +352,54 @@ class DuckDBType(SqlglotType):
     default_decimal_scale = 3
     default_interval_precision = "us"
 
+    @classmethod
+    def _from_sqlglot_TIMESTAMP(cls) -> dt.Timestamp:
+        return dt.Timestamp(scale=6, nullable=cls.default_nullable)
+
+    @classmethod
+    def _from_sqlglot_TIMESTAMPTZ(cls) -> dt.Timestamp:
+        return dt.Timestamp(scale=6, timezone="UTC", nullable=cls.default_nullable)
+
+    @classmethod
+    def _from_sqlglot_TIMESTAMP_S(cls) -> dt.Timestamp:
+        return dt.Timestamp(scale=0, nullable=cls.default_nullable)
+
+    @classmethod
+    def _from_sqlglot_TIMESTAMP_MS(cls) -> dt.Timestamp:
+        return dt.Timestamp(scale=3, nullable=cls.default_nullable)
+
+    @classmethod
+    def _from_sqlglot_TIMESTAMP_NS(cls) -> dt.Timestamp:
+        return dt.Timestamp(scale=9, nullable=cls.default_nullable)
+
+    @classmethod
+    def _from_ibis_Timestamp(cls, dtype: dt.Timestamp) -> sge.DataType:
```

Comment:
Lemme see if we actually ever hit this.

Reply:
Ah, this won't ever be hit. We don't have any APIs that are currently turning Ibis types into sqlglot types.

In the future, `create_table` will need this. I'll remove this here and add it in a follow-up to move duckdb backend DDL to use sqlglot.w